corehq/apps/accounting/migrations/0039_auto_20180730_1449.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2018-07-30 14:49
+from __future__ import absolute_import, unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounting', '0038_auto_20180730_1401'),
+    ]
+
+    operations = [
```

Comment:
Generally unless I want to be able to run the migrations one at a time, I'd squash them into one per PR.  I'd use ```squashmigrations``` if there's no ```HqRunPython``` involved.